src/actions/getSummonerAction.ts

```diff
@@ -12,31 +12,28 @@ export const getSummoner = (summonerId: string, region: string) => async (dispat
     dispatch({ type: GET_SUMMONER_REQUEST });
     try {
         const { data } = await Axios.get(`${TEST_BASE}/summonorById/proxy/${summonerId}/${region}`);
-        console.log('data ==>> ', data)
         dispatch({ type: GET_SUMMONER_SUCCESS, payload: data });
         localStorage.setItem(USER_ID, data.id);
         const id: string = data.id;
         const accountId: string = data.accountId;
-        console.log('id', id)
-        console.log('accountId', accountId)
 
         // summoner 기본 정보 받았으니 => summoner 디테일 뽑기
         try {
             const { data } = await Axios.get(`${TEST_BASE}/summonorById/proxy/${id}/${region}/summonerDetail`);
-            const detail: SummonerDetailType = data;
+            const detail: SummonerDetailType = data[0];
             dispatch({ type: GET_SUMMONER_DETAIL_SUCCESS, payload: detail });
 
             // 해당유저의 게임 했던것들 정보 가져옴 총 100개
             try {
                 const { data } = await Axios.get(`${TEST_BASE}/summonorById/proxy/${accountId}/${region}/matchId`);
                 const matches: MatchType[] = data.matches;
 
+                // 게임 id(number)만 뽑아서 길이가 100인 배열을 만든다. 
                 const matchIds = matches.reduce((a: number[], c) => {
                     a.push(c.gameId)
                     return a;
                 }, [])
```

Comment:
```suggestion
                const matchIds = matches.map(({ gameId }) => gameId);
```

굳이 `reduce`안써도 맵으로 돌리면 될거같네여 읽기도 좀 쉽고